src/os_win/os_errno.c

```diff
@@ -32,11 +33,28 @@ __wt_map_error_to_windows_error(int error) {
  *	Return a positive integer, a decoded Windows error
  */
 static int
-__wt_map_windows_error_to_error(DWORD winerr) {
+__wt_map_windows_error_to_error(DWORD winerr)
+{
 	return (winerr + windows_error_offset);
 }
 
 /*
+ * __wt_map_error_rdonly --
```

Comment:
I think that is a larger project with different requirements, potentially.  Readonly needs to detect two very specific situations.  I'd rather not delay this work for a general windows error discussion.  Can we create a new ticket for that?  It isn't clear to me that translating all errors to posix is what a Windows programmer would expect.